INSTALL.md

```diff
@@ -164,10 +164,28 @@ Some of our examples will also need **libSplash**.
       [LD\_LIBRARY\_PATH](#additional-required-environment-variables-for-optional-libraries)
       to `export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$ADIOS_ROOT/lib`
 
-- **ISAAC** (requires *boost mpl*, *boost fusion*, *IceT*, *Jansson*, *libjpeg*)
-    - Enables live in situ visualization, see more
-      - [General installation informations](https://github.com/ComputationalRadiationPhysics/isaac/blob/master/INSTALL.md)
-      - [PIConGPU specific installation informations](https://github.com/ComputationalRadiationPhysics/picongpu/wiki/Plugin%3A-ISAAC)
+- **ISAAC** (requires *boost* (header only), *IceT*, *Jansson*, *libjpeg*)
```

Comment:
Imho the only reason, why the listing of dependencies makes sense here at all, is to know, what you have to be installed, so that `$PICSRC/configure` enables the plugin/feature.
You don't need libwebsockets for this case.
But if you insist, I will add it, I honestly do not *really* care.